packages/ddex/ingester/parser/parser.go

```diff
@@ -101,7 +106,7 @@ func (p *Parser) processDelivery(changeStream *mongo.ChangeStream) {
 				"upload_etag":          delivery.UploadETag,
 				"delivery_id":          delivery.ID,
 				"create_track_release": track,
-				"publish_date":         track.Metadata.ReleaseDate, // TODO: Use time instead of string so it can be queried properly
+				"publish_date":         track.Metadata.ReleaseDate,
```

Comment:
removing this TODO because i completed it [here](https://github.com/AudiusProject/audius-protocol/commit/d7b925967062074e149cf617d9ad70695d166203#diff-1356b0db669dbfcb7d9cae7ebce00fc3930025081efc26fe9b995049c27cabb6R174-R184)